Dockerfile.template

```diff
@@ -149,6 +155,7 @@ RUN set -eux; \
 		| sort -u \
 		| xargs -r apt-mark manual \
 	; \
+	apt-mark manual librbd1; \
```

Comment:
When we add `--enable-rbd`, doesn't _something_ in QEMU end up linking to `librbd.so` such that this extra `apt-mark manual` isn't necessary?

Reply:
I've checked now and:
```
[...]
rbd support       yes
[...]

[...]
Removing librbd-dev (15.2.4-1~bpo10+1) ...
Removing librbd1 (15.2.4-1~bpo10+1) ...
Removing librados-dev (15.2.4-1~bpo10+1) ...
Removing librados2 (15.2.4-1~bpo10+1) ...
[...]
```
By manually marking `librbd1`, both `librbd1` and `librados2` are kept.